### PR TITLE
fixes for pbss replay

### PR DESCRIPTION
**Pull request by gballet**

### CODE REVIEW

File: trie/verkle.go [hunk 1]
```diff
@@ -257,13 +257,15 @@ func (trie *VerkleTrie) Commit(_ bool) (common.Hash, *NodeSet, error) {
...
 	for _, node := range nodes {
-		if err := trie.db.diskdb.Put(append([]byte("flat-"), node.Path...), node.SerializedBytes); err != nil {
+		if err := batch.Put(append([]byte("flat-"), node.Path...), node.SerializedBytes); err != nil {
 			return common.Hash{}, nil, fmt.Errorf("put node to disk: %s", err)
 		}
+		batch.Write()
 	}
 
-	return nodes[0].CommitmentBytes, nil, nil
+	return common.BytesToHash(nodes[0].CommitmentBytes[:32]), nil, nil
```
Comment (gballet) [inline trie/verkle.go:268]: Note to self: this is incorrect, we will need to perform a serialization here... or sha2 the coordinates?

File: cmd/utils/cmd.go [hunk 2]
```diff
@@ -173,6 +174,17 @@ func ImportChain(chain *core.BlockChain, fn string) error {
 			return err
 		}
 	}
+	cpuProfile, err := os.Create("cpu.out")
+	if err != nil {
+		return fmt.Errorf("Error creating CPU profile: %v", err)
```
Comment (jsign) [inline cmd/utils/cmd.go:179]: Mind switching to lower case (i.e: `error` not `Error`). Same with L184

File: go.mod [hunk 3]
```diff
@@ -13,7 +13,7 @@ require (
 	github.com/cespare/cp v0.1.0
 	github.com/cloudflare/cloudflare-go v0.14.0
 	github.com/consensys/gnark-crypto v0.4.1-0.20210426202927-39ac3d4b3f1f
-	github.com/crate-crypto/go-ipa v0.0.0-20230601170251-1830d0757c80
+	github.com/crate-crypto/go-ipa v0.0.0-20230626131944-6a9b06cf26df
```
Comment (jsign) [inline go.mod:16]: @gballet, should we target https://github.com/gballet/go-verkle/pull/354 this now, right?

File: trie/verkle.go (hunk 1 above)
Comment (jsign) [reply trie/verkle.go:278]: Is this a TODO that is worth adding in the code? Or this should be fixed before merging?

File: go.mod (hunk 3 above)
Comment (gballet) [reply go.mod:16]: yes

File: trie/verkle.go (hunk 1 above)
Comment (gballet) [reply trie/verkle.go:278]: nope, it needs to be fixed before we merge, although it can be fixed at the parent branch level

**Pull request merged**